tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/Request.cs

```diff
@@ -29,9 +29,16 @@ public async Task PutJsonAsync(string url, object payload, params object[] pathS
 
         private IFlurlRequest CreateCommonRequest(string url, params object[] pathSegments)
         {
+            string accessToken = null;
+
+            if (_context.ContainsKey(ScenarioContextKeys.AccessToken))
+            {
+                accessToken = _context[ScenarioContextKeys.AccessToken] as string;
```

Comment:
Would you consider using ScenarioContexts method 'Get<T>'?  

`accessToken = _context[ScenarioContextKeys.AccessToken] as string;` -> `accessToken = _context.Get<string>(ScenarioContextKeys.AccessToken);`

Reply:
I would indeed!

Reply:
👍